code/modules/mob/mob.dm

```diff
@@ -486,9 +486,27 @@
 		// shift-click catcher may issue examinate() calls for out-of-sight turfs
 		return
 
-	if(is_blind(src) && !blind_examine_check(A))
+	var/isblind = is_blind(src)
+	if(isblind && !blind_examine_check(A))
 		return
 
+	if(!isobserver(usr) && !(usr == A) && !isblind)
+		var/list/can_see_target = viewers(usr)
+		for(var/mob/M as anything in viewers(4, usr))
+			if(!M.client)
+				continue
+
+			if(isitem(A))
+				var/obj/item/I = A
+				if((I.item_flags & IN_STORAGE) && get(I, /mob/living) == src)
```

Comment:
`get()`? what's this for?

Reply:
Checks if said item is being held by the mob (or in this case, backpack) so it prints that text

Reply:
I just realized I forgot the other bit since this commit was rushed actually, but this is what it should actually look when examining an item inside your backpack compared to an item inside a backpack on the ground
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/66234359/5995df82-c3a4-49dc-86b5-c31e4e258f46)